### ISSUE TITLE
Reset Sliders does not work with Parallax?

### ISSUE THREAD

**Issue by arixv** [alvarotrigo/fullPage.js#2993]
### Description

Hi Alvaro, we've just bought the ResetSlider extension and integrated with our Site.
We're also using Parallax for Each section and they have a 'fp-bg' with background images.
When the slide is reseted, the background is set in the middle of the screen.

Maybe it is an incompatibility between both extensions?

Here it's our test website: [removed link]

Thank you for your help

**Comment by alvarotrigo** [comment]
There's indeed an issue there. Mostly when using `type: 'cover'` instead of the default `type: 'reveal'`.

Thanks for reporting it! I'll check it out! 
If you want a refund and prefer not to wait for the fix please let me know!

**Comment by alvarotrigo** [comment]
I've fixed the issue. 
Please [contact me](https://alvarotrigo.com/#contact) with the email you used for the purchase and I'll send you the new versions of both extension, parallax and reset sliders.

**Comment by tschortsch22** [comment]
Hi, same issue occurs using parallax and interlocked slides extension with following structure:

```
section 1
section 2 with 3 slides, slide 2 set to active
section 3 with 3 slides
section 4 with 3 slides
section 5 with 3 slides
```

When scrolling down section 1 & 2 are fine, section 3, 4 & 5 only show 50 % of the image. Sliding left/right and back again recovers image with full width.

**Comment by alvarotrigo** [comment]
@tschortsch22 the issue was solved in the interlocked extensions version 0.0.3. Please [contact me](https://alvarotrigo.com/#contact) with the email you used for the purchase and I'll send you the new versions of the extension.
